libs/shared/flagd-js-core/src/lib/feature-flag.ts

```diff
@@ -0,0 +1,202 @@
+import {FlagValue} from '@openfeature/server-sdk';
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+/**
+ * Flagd flag configuration structure.
+ */
+export class FeatureFlag {
+  private readonly _state: string;
+  private readonly _defaultVariant: string;
+  private readonly _variants: Map<string, FlagValue>;
+  private readonly _targetingString: string;
+
+  constructor(flag: any) {
```

Comment:
Please replace `any` with a type that represents a flagd configuration.

Reply:
Changed upstream, please check PR - https://github.com/open-feature/js-sdk-contrib/pull/620